src/main/java/com/xebialabs/deployit/ci/DeployitNotifier.java

```diff
@@ -49,6 +50,8 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import javax.ws.rs.POST;
```

Comment:
Wrong annotation, it's in `org.kohsuke.stapler` somewhere. If your core dependency is too low, use `RequirePOST` instead.

Reply:
Removed @POST. 
Implemented form validation by checking permissions